functional/platform/nspawn.go

```diff
@@ -384,14 +384,14 @@ func (nc *nspawnCluster) CreateMember() (m Member, err error) {
 	return nc.createMember(id)
 }
 
-func newMachineID() string {
+func (nc *nspawnCluster) NewMachineID() string {
```

Comment:
Oh, right, this *might* be specific to a certain cluster type... I was just thinking of exporting is as a function from where it is, but that obviously would conflict with other implementations. Sorry for the noise.